README.md

```diff
@@ -42,25 +51,20 @@ It will output the `.vsix` file to:
 
 #### Using the Extension with Changes
-
 Once you have the Visual Studio environment with the modified extension loaded, or have installed the extension from the `.vsix` file, you can test it by opening the example project:
 
 	Conan.VisualStudio.Examples/ExampleCLI/ExampleCLI.sln
 
-This example has a `conanfile.txt` with a dependency on the popular formatting library `fmt`.  When the project is first opened, Visual Studio is unable to find the `fmt` project dependency.  The extension should immediately run however, and Visual Studio should give you a toolbar dropdown message offering to refresh the project.  After refreshing, the `fmt` headers should be found by the compiler, and the `lib` files should be found by the linker successfully.  
-
-
-Update 03-14-2019
-------------------
-Thanks to @sboulema and @SSE4, the extension now provides the minimum required functionality for basic use and initial testing. 
-
-Update 02-06-2019
-------------------
-Plugin development is now being resumed after being stagnant for the past year. 
-
-The overall goal of the extension is for Visual Studio to be able to execute Conan automatically as-needed based on the currently loaded solution/project/configuration.  Over time, this could grow to a lot of convenience operations.  However, the primary (first) objective is to run `conan install` which will generate `conanbuildinfo.props` and satisfy the dependencies. 
+This example has a `conanfile.txt` with a dependency on the popular formatting library `fmt`. When the project is first opened, Visual Studio is unable to find the `fmt` project dependency. The extension should immediately run however, and Visual Studio should give you a toolbar dropdown message offering to refresh the project. After refreshing, the `fmt` headers should be found by the compiler, and the `lib` files should be found by the linker successfully.  
 
-Thus, the first requirement is to provide users a mechanism for mapping each Solution/Project/Configuration to a corresponding `conan install` command. A strategy has been chosen for this, and is being discussed here: https://github.com/conan-io/conan-vs-extension/issues/5
+Changelog
+---------
 
-Overall, future features and should try to use a similar configuration-file-based strategy to provide maximum configurability and flexibility to the user of the extension, by making any new conan-related-operations exposed in any toolbar menus and right-click menus configurable and composeable.  This is particular important in the near-term while we are still deciding how the parts should work together, so that new workflow ideas can be tested without requiring code changes and rebuilds of the extension. 
+This extension started was started by community members like @sboullema, @SSE4, @ForNeVer and @solvingj, thanks to
```

Comment:
I don't think it's useful information for the changelog, we may just omit it or move somewhere else (below or to contributors file)